generic/generic_cmd-1.2.3.recipe

```diff
@@ -17,7 +17,7 @@ SOURCE_DIR="$portVersionedName"
 PATCHES="projectx-$portVersion.patchset"
 ADDITIONAL_FILES="projectx.rdef"
 
-ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
```

Comment:
Shouldn't we remove x86 here (and in other places), there is no primary architecture for that anymore (afaik)

Reply:
should update to show new use of 'all'